### PR TITLE
Fix cron

### LINKED ISSUE #100: bug: cron job doesn't work for private website
During installation, if you answer "No" to "Is it a public website ?" question, your WordPress blog is only visible for YunoHost users.

Few days after installation, you got this message on WordPress panel:

> L’évènement planifié wp_scheduled_delete a échoué. Votre site fonctionne toujours, mais cela pourrait indiquer que les publications planifiées ou les mises à jour automatiques ne fonctionnent pas comme elles le devraient.

The cron job installed on YunoHost server doesn't run correctly each night at 03:00 because website is not reachable without a YunoHost authentication:

```
# Reach everyday wp-cron.php?doing_wp_cron to trig the internal wordpress cron.
0 3  *  *  * root wget -q -O - https://domain.fr/blog/wp-cron.php?doing_wp_cron >/dev/null 2>&1
```

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *If the app is private, the http request fails*

## Solution
- *Do not use http request but call the file directly (Fix https://github.com/YunoHost-Apps/wordpress_ynh/issues/100)*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/wordpress_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/actions/reset_default_app [hunk 1]
```diff
@@ -66,8 +66,8 @@ yunohost app action run $app reset_default_phpfpm
 # CREATE A CRON TASK FOR AUTOMATIC UPDATE
 #=================================================
 
-echo "# Reach everyday wp-cron.php?doing_wp_cron to trig the internal wordpress cron.
-0 3  *  *  * root wget -q -O - https://$domain$path_url/wp-cron.php?doing_wp_cron >/dev/null 2>&1" > /etc/cron.d/$app
+echo "# Reach everyday wp-cron.php to trig the internal WordPress cron.
+0 3  *  *  * $app php$phpversion  $final_path/wp-cron.php" > /etc/cron.d/$app
```
Comment (alexAubin) [inline scripts/actions/reset_default_app:70]: Shouldn't we remove that trailing quote after php at the end ?

File: scripts/install [hunk 2]
```diff
@@ -228,8 +228,8 @@ ynh_store_file_checksum --file="$final_path/wp-config.php"
 # CREATE A CRON TASK FOR AUTOMATIC UPDATE
 #=================================================
 
-echo "# Reach everyday wp-cron.php?doing_wp_cron to trig the internal WordPress cron.
-0 3  *  *  * root wget -q -O - https://$domain$path_url/wp-cron.php?doing_wp_cron >/dev/null 2>&1" > /etc/cron.d/$app
+echo "# Reach everyday wp-cron.php to trig the internal WordPress cron.
+0 3  *  *  * $app php$phpversion  $final_path/wp-cron.php" > /etc/cron.d/$app
```
Comment (alexAubin) [inline scripts/install:232]: (same here)

File: scripts/upgrade [hunk 3]
```diff
@@ -263,8 +263,8 @@ ynh_store_file_checksum --file="$final_path/wp-config.php"
 # CREATE A CRON TASK FOR AUTOMATIC UPDATE
 #=================================================
 
-echo "# Reach everyday wp-cron.php?doing_wp_cron to trig the internal wordpress cron.
-0 3  *  *  * root wget -q -O - https://$domain$path_url/wp-cron.php?doing_wp_cron >/dev/null 2>&1" > /etc/cron.d/$app
+echo "# Reach everyday wp-cron.php to trig the internal WordPress cron.
+0 3  *  *  * $app php$phpversion  $final_path/wp-cron.php" > /etc/cron.d/$app
```
Comment (alexAubin) [inline scripts/upgrade:267]: (same here)

**Review by alexAubin — APPROVED** [review]
(Otherwise LGTM :+1: )

File: scripts/actions/reset_default_app (hunk 1 above)
Comment (kay0u) [reply scripts/actions/reset_default_app:70]: The quote you see is open on the previous line, `echo "...`

Comment (alexAubin) [reply scripts/actions/reset_default_app:70]: :facepalm: Sorry for the noise

**Review by alexAubin — APPROVED** [review]
:+1:

**Pull request merged**